cipd_packages/ruby/ruby_ship/tools/ruby_ship_build.sh

```diff
@@ -0,0 +1,264 @@
+#!/usr/bin/env bash
+# Copyright 2023 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+set -x
+#VERIFY PARAMETERS
+if [ $# -eq 0 ]
+  then
+    echo "No arguments supplied"
+    echo "Usage: nix_compile_ruby.sh /path/to/ruby_source.tar.gz"
+    exit 1
+fi
+
+#DETECT OPERATING SYSTEM
+OS="unknown"
+if [[ "$OSTYPE" == "linux"* ]]; then
```

Comment:
nit: Google style guide for .sh vars is to use {}.

Reply:
obsolete, this logic was removed.